### DISCUSSION TITLE
On android, on super notes, the formatting bar is below the android bar making it difficult to use

### DISCUSSION THREAD

**Discussion by jrcacd** [standardnotes/forum#2905] [General]
![example](https://user-images.githubusercontent.com/26005662/203743603-93a54add-1e16-45fc-818b-d8cf1b9a6494.jpg)

**Comment by myreli** [comment]
Hi @jrcacd! Now on mobile, instead of the inline toolbar, Super offers a fixed toolbar at the bottom:

<img width="394" alt="image" src="https://user-images.githubusercontent.com/17554234/221391053-b3685e31-272d-4d4e-bc13-f3a6f6e51cbc.png">
